Explain how users are backed up and restored (#13081)
crate/crate: docs/admin/user-management.rst

When CrateDB is started, the cluster contains one predefined superuser. This
user is called ``crate``. It is not possible to create any other superusers

> Users\' information is backed up together with the cluster\'s metadata when a snapshot is created, and it can be restored independently with the ``USERS`` option of the :ref:`sql-restore-snapshot` command.

Reviewer: I think a backslash for escaping the apostrophe isn't needed here, it also compiles without.

And you might want to add a line break as per https://github.com/crate/crate/blob/master/devs/docs/write_docs.rst:
> Line length must not exceed 79 characters (except for literals that cannot be wrapped)

When CrateDB is started, the cluster contains one predefined superuser. This
user is called ``crate``. It is not possible to create any other superusers

> Users' information is backed up together with the cluster's metadata when a

Reviewer 2: ```suggestion
User's information is backed up together with the cluster's metadata when a
```

Author: Hi Sebastian, it was meant as the information about users plural, but maybe it would be clearer if I reword it to avoid the ' altogether? I could maybe also explain that a restore only creates missing users it does not delete users that did not exist at the time of the backup, what do you think?

> together with the cluster's metadata when a snapshot is created, and it can be
> restored with the ``USERS`` option of the :ref:`sql-restore-snapshot` command.

Reviewer 2: It will be also restored when using `RESTORE ALL`, see https://crate.io/docs/crate/reference/en/5.0/sql/statements/restore-snapshot.html. `RESTORE USERS` (or any other section) will restore the users **only**.
```suggestion
together with the cluster's metadata when a snapshot is created, and is 
restored when using the ``ALL`` or ``USERS`` clause of the 
:ref:`sql-restore-snapshot` command.
```
